scalacheck/test/src/peschke/scalacheck/NumericRangeGensLongTest.scala

```diff
@@ -2,13 +2,12 @@ package peschke.scalacheck
 
 import org.scalacheck.Gen
 import org.scalatest.Inspectors
-import org.scalatest.prop.TableDrivenPropertyChecks
 import peschke.PropSpec
 import peschke.scalacheck.NumericRangeGensLongTest.PrettyRange
 
 import scala.collection.immutable.NumericRange
 
-class NumericRangeGensLongTest extends PropSpec with TableDrivenPropertyChecks {
```

Comment:
I have always frowned upon `TableDrivenPropertyChecks`, Maybe time to reconsider.

**`[commentary]`**

Reply:
They definitely came in handy, much easier to use than defining a limited `Gen` or using `Seq(...).foreach` to DIY it.